packages/tools/client-debugger/client-debugger-view/src/components/TreeRenderHelper.tsx

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, VisualNode, VisualNodeKind } from "@fluid-tools/client-debugger";
+// eslint-disable-next-line import/no-internal-modules
+import { Tree, TreeItem, TreeItemLayout } from "@fluentui/react-components/unstable";
+import { Divider } from "@fluentui/react-components";
+import { TreeDataView } from "./TreeDataView";
+
+/**
+ * TODO
+ */
+export interface TreeRenderHelperProps extends HasContainerId {
+	/**
+	 * TODO
+	 */
+	containerId: string;
```

Comment:
Since you're already extending `HasContainerId`, you shouldn't need this.

Reply:
You will want to extend the React HasChildren base interface though, so you can pass the child elements down for recursive rendering.

Reply:
Extends `PropsWithChildren` in `RenderSummaryTree` component.